fixtures/golden/codegen/python/fastapi/postgres/url_shortener/app/main.py

```
@@ -0,0 +1,54 @@
+    ],
+    allow_credentials=True,
+    allow_methods=["*"],
+    allow_headers=["*"],
+)
+
+
+app.include_router(url_mappings.router)
+
+
+
+import os as _os
```

Ruff (0.15.12)

[error] Module level import not at top of file

(E402)